docs/tutorials/kubectl/app-management-overview.md

```diff
@@ -0,0 +1,238 @@
+---
+title: App Management Overview
+---
+
+{% capture overview %}
+
+*Kubectl* supports several different approaches for creating and managing
+applications in a Kubernetes cluster.  This document provides an overview
+of the different approaches and which environment each approach is best
+suited towards.
+
+{% endcapture %}
+
+{% capture prerequisites %}
+
+{% include task-tutorial-prereqs.md %}
+
+### Prerequisites
+
+- [Kubernetes Basics Tutorial](/docs/tutorials/kubernetes-basics/)
+
+{% endcapture %}
+
+{% capture objectives %}
+
+- Explore the different approaches for managing applications supported by *kubectl*.
+- Explore which environment each approach is best suited for.
+
+{% endcapture %}
+
+
+{% capture lessoncontent %}
+
+# App Management Styles Table
+
+**Currently, a given resource should be managed using only 1 style.  Attempting
+to mix-and-match styles for the same resource will result in undefined behavior.**
+
+| Management Style          | Environment             | Writers  | Operates Best On     | Learning Curve |
+|---------------------------|-------------------------|----------|----------------------|----------------|
+| Imperative Commands       | Dev or Small Clusters   | 1+       | Live Resources       | Lowest         |
+| Basic Yaml Config         | Production Clusters     | 1        | Individual Files     | Moderate       |
+| Yaml Config with Apply    | Production Clusters     | 1+       | Directories of Files | Highest        |
+
+# Imperative Commands
+
+## Command Examples
+
+Run an instance of the *nginx* container by creating a Deployment Resource
+
+```sh
+kubectl run nginx --image nginx
+```
+
+Same as above with different syntax (specifies resource type)
+
+```sh
+kubectl create deployment nginx --image nginx
+```
+
+## Command Overview
```

Comment:
This isn't a single command but rather a set.  Perhaps list the imperative commands directly here?